src/views/Home/components/Schedule/components/index.scss

```diff
@@ -0,0 +1,118 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.schedule-table {
+  position: relative;
+  width: 100%;
+
+  &__table {
+    min-height: 300px;
```

Comment:
From the figma, it seems like the table should have the same height for each component and the min-height should be a bit higher on desktop